BlazorApp1/Program.cs

```diff
@@ -1,11 +1,11 @@
-using BlazorApp1.Data;
+using BlazorApp1.Services;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 builder.Services.AddRazorPages();
 builder.Services.AddServerSideBlazor();
-builder.Services.AddSingleton<WeatherForecastService>();
+builder.Services.AddSingleton<VideoAnnoatationService>();
```

Comment:
@WanjiruMambo @CKanja  If you create a Service ( kind of like a module that is reusable) you have to register it here like I did.